lando_cli/cli.py

```diff
@@ -272,20 +293,26 @@ def get_new_commits(local_branch: str, base_sha: str, repo: Path) -> list[str]:
     return commits
 
 
-def get_commit_patches(commits: list[str], repo: Path) -> list[str]:
+def get_commit_patches(commits: list[str], repo: Path) -> list[bytes]:
     """Get `git format-patch` patches for each passed commit SHA."""
     patches = []
     for idx, commit in enumerate(commits):
-        patch = git_run(["format-patch", commit, "-1", "--always", "--stdout"], repo)
+        patch = git_run_bytes(
+            ["format-patch", commit, "-1", "--always", "--stdout"], repo
+        )
         patches.append(patch)
 
     return patches
 
 
-def create_add_commit_actions(patches: list[str]) -> list[dict]:
-    """Given an ordered list of patches, create `add-commit` actions for each."""
+def create_add_commit_actions(patches: list[bytes]) -> list[dict]:
+    """Given an ordered list of patches, create `add-commit-base64` actions for each."""
     return [
-        {"action": "add-commit", "content": patch, "patch_format": "git-format-patch"}
+        {
+            "action": "add-commit-base64",
+            "content": base64.b64encode(patch),
```

Comment:
`b64encode` still returns `bytes`, we need to convert to an ASCII string to submit as JSON. In `mach try` we have this helper:

```python
def convert_bytes_patch_to_base64(patch_bytes: bytes) -> str:
    """Return a base64 encoded `str` representing the passed `bytes` patch."""
    return base64.b64encode(patch_bytes).decode("ascii")
```